site/content/news/rdf4j-411.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "RDF4J 4.1.1 and 4.0.5 released"
+date: 2022-09-01T14:51:16+0200
+layout: "single"
+categories: ["news"]
+---
+RDF4J 4.1.1 and 4.0.5 are now available. RDF4J 4.1.1 is a patch release 
+fixing 13 bugs and 1 security vulnerability (CVE-2020-36518 in FasterXML 
+Jackson Databind). Two of the issues fixed in 4.1.1, including the patch for security vulnerability, have been back ported and releases as 4.0.5.
+
+For more details, have a look at the [4.1.1 release notes](/release-notes/4.1.1) and [4.0.5 release notes](/release-notes/4.0.5).
+<!--more-->
+
+
+
+### Links
+
+- [Download RDF4J](/download/)
+- [4.1.1 release notes](/release-notes/4.1.1)
+- [4.0.5 release notes](/release-notes/4.0.5)
```

Comment:
@jeenbroekstra 

I've release 4.1.1 and also back ported the fix for clearing the default graph together with the jackson upgrade as 4.0.5.

Reply:
Cool - nice work. Was the backport requested by someone in particular?

Reply:
Ontotext wanted the fix for deleting the default graph back ported since they need to implement it server side in GraphDB.